node/src/components/consensus/highway_core/active_validator.rs

```diff
@@ -91,12 +90,12 @@ impl<C: Context> ActiveValidator<C> {
 
     /// Returns actions a validator needs to take at the specified `timestamp`, with the given
     /// protocol `state`.
-    pub(crate) fn handle_timer<R: Rng + CryptoRng + ?Sized>(
+    pub(crate) fn handle_timer(
         &mut self,
         timestamp: Timestamp,
         state: &State<C>,
         instance_id: C::InstanceId,
-        rng: &mut R,
+        rng: &mut dyn CryptoRngCore,
```

Comment:
I'm fine with this; just out of curiosity: Is there a reason to make this change in non-trait methods as well?

Reply:
Mainly for consistency. Otherwise one has to pass what is essentially a `&mut &mut R`.